src/libs/KeyboardShortcut/index.js

```diff
@@ -1,6 +1,23 @@
 import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import getOperatingSystem from '../getOperatingSystem';
+import CONST from '../../CONST';
 
 const events = {};
+const keyboardShortcutMap = {};
+
+/**
+ * Return the key-value pair for shortcut keys and translate keys
+ * @returns {Array}
+ */
+function getKeyboardShortcuts() {
+    const shortcuts = [];
+    _.each(keyboardShortcutMap, (descriptionKey, key) => {
+        shortcuts.push({key, descriptionKey});
```

Comment:
Just noticing a subtle inconsistency here in that in `CONST.js` we define keyboard shortcuts as having the following shape:

```js
{
    shortcutKey: String,
    descriptionKey: String,
    modifiers: Array<String>,
}
```

But here we're defining them with this a slightly different shape:

```js
{
    key: String, // this is really a string with modifiers + the key,
    descriptionKey: String
}
```

So I find  this similar but separate representation a bit confusing, and I think we could make this a bit clearer by having `keyboardShortcutMap` store a representation of keyboard shortcuts with the following shape:

```js
{
    // The key that is pressed to initiate some action
    shortcutKey: String,
    
    // Modifier keys that also must be pressed with the shortcut key to initiate this action
    modifiers: Array<String>,
    
    // The human-readable representation of the shortcut with its modifiers (i.e: `CMD + SHIFT + K`)
    displayName: String,

    // A translation key for the full description of the action initiated by this keyboard shortcut    
    descriptionKey: String,
}
```